milestone-3/frontend/src/components/recipePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+interface Recipe {
+    name: string;
+    image: string;
+    desc: string;
+}
+interface RecipePreviewProps extends Recipe {
+    external?: boolean;
+  }
```

Comment:
because the Recipe interface above is only used here you could have combined it with this instead of extending it like you did.